RealmSwift/RealmKeyedCollection.swift

```diff
@@ -279,6 +278,125 @@ public protocol RealmKeyedCollection: Sequence, ThreadConfined, CustomStringConv
                  _ block: @escaping (RealmMapChange<Self>) -> Void)
     -> NotificationToken
 
+    /**
```

Comment:
A dictionary-specific thing that should be noted is that keypaths can only be properties and not keys in the dictionary.

Reply:
See addition of :
```
     - note: The keyPaths parameter refers to object properties of the collection type and
             *does not* refer to particular key/value pairs within the Map.
```
I decided to put this underneath the `-parameter keyPaths` tag because it should be easily visible (though I get that the "parameter" section here is getting bloated).